projects/lib/src/tournament.cpp

```diff
@@ -250,6 +250,15 @@ void Tournament::setPgnCleanupEnabled(bool enabled)
 	m_pgnCleanup = enabled;
 }
 
+void Tournament::setEpdOutput(const QString& fileName)
+{
+	if (fileName != m_epdFile.fileName())
+	{
+		m_epdFile.close();
```

Comment:
Probably the file handle should be closed in the destructor of `Tournament`. And the same goes for `m_pgnFile` which I've apparently forgotten to close.